sinusbot_installer.sh

```diff
@@ -927,7 +924,7 @@ if [ "$YT" == "Yes" ]; then
     redMessage "Cronjob already set for YT-DL updater"!
   else
     greenMessage "Installing Cronjob for automatic YT-DL update..."
-    echo "0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename >/dev/null" >>/etc/cron.d/ytdl
+    printf "SHELL=/bin/sh\nPATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin\n0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl
```

Comment:
Why are you setting there the `PATH` environment variables for this cronjob manually?
Instead I would only set an absolute path for the youtube-dl binary.